docs/devElements.md

```diff
@@ -125,7 +125,7 @@ configure them for your element as follows:
     "controller": "pl-my-element.py",
     "dependencies": {
         "coreScripts": [
-            "d3.min.js"
+            "PrairieUtil.js"
```

Comment:
How about we just remove `coreScripts` from the example completely, so that people are less likely to copy it?

Reply:
Done.